Debugger breaks in next script after stepOut from previous one.

Running test: testStepOut
test (foo.js:12:2)
(anonymous) (:0:0)

(anonymous) (:0:5)

(anonymous) (timeout1.js:0:0)

foo (timeout2.js:1:12)

(anonymous) (timeout3.js:0:0)


Running test: testStepOver
(anonymous) (:0:0)

test (foo.js:12:2)
(anonymous) (:0:0)

test (foo.js:13:0)
(anonymous) (:0:0)

(anonymous) (:0:5)

(anonymous) (timeout1.js:0:0)

(anonymous) (timeout1.js:0:8)

(anonymous) (timeout1.js:0:34)

foo (timeout2.js:1:12)

foo (timeout2.js:2:2)

foo (timeout2.js:3:0)

(anonymous) (timeout3.js:0:0)

(anonymous) (timeout3.js:0:8)

(anonymous) (timeout3.js:0:34)


Running test: testStepInto
(anonymous) (:0:0)

test (foo.js:9:2)
(anonymous) (:0:0)

test (foo.js:10:2)
(anonymous) (:0:0)

test (foo.js:11:2)
(anonymous) (:0:0)

test (foo.js:12:2)
(anonymous) (:0:0)

test (foo.js:13:0)
(anonymous) (:0:0)

(anonymous) (:0:5)

(anonymous) (timeout1.js:0:0)

(anonymous) (timeout1.js:0:8)

(anonymous) (timeout1.js:0:34)

foo (timeout2.js:1:12)

foo (timeout2.js:2:2)

foo (timeout2.js:3:0)

(anonymous) (timeout3.js:0:0)

(anonymous) (timeout3.js:0:8)

(anonymous) (timeout3.js:0:34)

